packages/bpk-component-calendar/src/BpkCalendarDate.js

```diff
@@ -139,7 +139,8 @@ class BpkCalendarDate extends PureComponent {
         type="button"
         style={style}
         className={classNames.join(' ')}
-        aria-label={date.getDate()}
+        aria-hidden={isBlocked}
+        aria-label={isBlocked ? '' : date.getDate()}
```

Comment:
Date change 1 - added here as date can be used as its own component.

Reply:
Is this change needed if we have `aria-hidden`?

Reply:
Technically no - aria-hidden takes precedence. I added it since as it is hidden the aria-label is redundant, so we can avoid the 'work' of setting the label.

Reply:
> Is this really worth the extra code complexity?

No strong opinion, so have removed.